src/screens/LoginScreen/components/FormLogin/index.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+
+import VioletButton from '../../../../components/VioletButton';
+import WhiteButton from '../WhiteButton';
+import img from '../../assets/img/erroricon.png';
+import { sendGame } from '../../../../services/gameService';
+
+import styles from './index.module.scss';
+
+class FormLogin extends React.Component {
+  state = {
+    errorCode: false,
+    errorName: false,
+    errorServer: false,
+    code: '',
+    name: ''
+  };
+
+  handleSendGame = async data => {
+    const res = await sendGame(data);
+    if (res.status === 500) {
+      this.setState({ errorCode: true });
+    } else if (res.status === 404) {
+      this.setState({ errorServer: true });
+    } else {
+      console.log('redireccionamiento');
+    }
+  };
+
+  handleChange = event => {
+    const { value, name } = event.target;
+    this.setState({ [name]: value });
+    this.setState({ errorCode: false, errorName: false, errorServer: false });
+  };
+
+  handleSubmit = event => {
+    event.preventDefault();
+    const { name, code } = this.state;
+    if (name === '' && code === '') {
+      this.setState({ errorName: true, errorCode: true });
+    } else if (name === '') {
+      this.setState({ errorName: true });
+    } else if (code === '') {
+      this.setState({ errorCode: true });
+    } else {
+      const data = { name, code };
+      this.handleSendGame(data);
+    }
```

Comment:
Fijate que no es necesario hacer el primer chequeo ya que `name` y `code` no son excluyentes entre si y se pueden validar por separado (es más, es necesario validarlos por separado).
Abajo sugerí una validación para realizar el send, pero hay que tener en cuenta que nada más contempla que el caso de error en el que `name` o `code` sean vacíos.
```suggestion
    if (name === '') {
      this.setState({ errorName: true });
    };
    if (code === '') {
      this.setState({ errorCode: true });
    };
    if (name && code) {
      const data = { name, code };
      this.handleSendGame(data);
    };
```